src/app/loaders/express/parser.ts

```diff
@@ -14,6 +14,16 @@ const parserMiddlewares = () => {
   // processes that must be done before any other middlewares
   const preMiddlewareProcesses = [convertSnsMessageType]
 
+  const bodyParserCompatibility: RequestHandler = (req, res, next) => {
+    req.body = req.body || {}
```

Comment:
nit: using the null coalesce operator (`??`) might be safer:
- falsy values (e.g. `""`, `0`, `false`, `[]`) should probably be preserved
- the intention seems to be to replace `null` / `undefined` with `{}`

```suggestion
    req.body = req.body ?? {}
```

Reply:
Good catch!